.github/workflows/distcheck.yaml

```diff
@@ -9,6 +9,58 @@ env:
     libhwloc-dev
     make
 jobs:
+  al2build:
+    runs-on: codebuild-ghactions-al2-${{ github.run_id }}-${{ github.run_attempt }}
+    strategy:
+      matrix:
+        sdk:
+          - cuda
+        efainstaller:
+          - latest
+          - 1.32.0
+          - 1.31.0
+          - 1.30.0
+    name: al2/${{ matrix.sdk }}/efa@${{ matrix.efainstaller }}/distcheck
+    steps:
+      # note, do not bump to v4: https://github.com/actions/checkout/issues/1590
+      - uses: actions/checkout@v3
+      - name: Fetch and Install EFA Installer Dependencies
+        run: |
+          curl -O https://efa-installer.amazonaws.com/aws-efa-installer-${{ matrix.efainstaller }}.tar.gz
+          tar -xvf aws-efa-installer-*.tar.gz
+          cd aws-efa-installer/RPMS/ALINUX2/x86_64
+          find . | grep rpm$ | xargs sudo yum -y localinstall
```

Comment:
You've probably mentioned this before, but why not `efa_installer.sh`?

Reply:
The idea of using the efa installer came up on the initial distcheck job and I was advocating for using it because it was easier, but we changed it to build from source in review.

I initially tried to just call the efa installer script here, again, but it failed due to the codebuild environment being stripped down and lacking things like initrd tools:

https://github.com/aws/aws-ofi-nccl/actions/runs/9092045347/job/24987917737#step:3:471